### PR TITLE
terraform example code optimized

### PR DESCRIPTION
**Pull request by marifse**
### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by marifse** [comment]
@ocofaigh please review this too, and let me know if any comments. Thanks

**Comment by ocofaigh** [comment]
@maheshwarishikha Do you know if there was a specific reason why we did not have tests for the basic, hybrid and complete examples? I think there may of been a reason?

**Comment by maheshwarishikha** [comment]
> @maheshwarishikha Do you know if there was a specific reason why we did not have tests for the basic, hybrid and complete examples? I think there may of been a reason?

@ocofaigh 
Hybrid one - we cant test because of hsm connector. We used one HSM connecotr ID provided by HPCS team for our dev activities. 
Basic and fscloud tests the same.

File: tests/other_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		Region:       "us-south",
+	})
+
+	output, err := options.RunTestConsistency()
+
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunHybridExample(t *testing.T) {
```
Comment (ocofaigh) [inline tests/other_test.go]: As per https://github.com/terraform-ibm-modules/terraform-ibm-hpcs/pull/166#issuecomment-2006814770 we cannoy add a test for this example - please remove it

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@marifse see my comment.

@maheshwarishikha The basic example is calling the root level module, yet the fscloud is calling the fscloud submodule, so I propose we keep a test for the basic example in other_test.go

File: tests/other_test.go (hunk 1 above)
Comment (marifse) [reply tests/other_test.go]: @ocofaigh hybrid test removed

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**